test/extended/testdata/test-cli-debug.yaml

```diff
@@ -0,0 +1,69 @@
+kind: List
+apiVersion: v1
+items:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: busybox
+  spec:
+    tags:
+    - name: latest
+      from:
+        name: busybox
+        kind: DockerImage
+
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    name: busybox1
+  spec:
+    replicas: 0
+    selector:
+      deploymentconfig: busybox1
+    template:
+      metadata:
+        labels:
+          deploymentconfig: busybox1
+      spec:
+        containers:
+        - name: busybox
```

Comment:
@jim-minter sorry, you're totally correct, i mixed up the container and image field while i was reading this.